provisioning_templates/snippet/built.erb

```diff
@@ -0,0 +1,10 @@
+<%#
+kind: snippet
+name: built
+model: ProvisioningTemplate
+snippet: true
+-%>
+# exit build mode (try with curl, wget and busybox)
+/usr/bin/curl -o /dev/null --insecure --silent '<%= foreman_url('built') %>' || \
```

Comment:
what happens if user has both curl and wget installed? would it be better to first detect the available tool? something like https://github.com/theforeman/foreman_chef/blob/master/app/views/foreman/unattended/snippets/_chef_client_omnibus_bootstrap.erb#L8-L12

and even better it would be to have it in core renderer :-)

Reply:
Added detection. I see no reason polluting core with such a powerful ERB system we have.

Reply:
I have few, testing, updates (one day we'll want to add third tool), easier to call one macro in any template than writing these lines etc, note this is not the only place where we use wget/curl